File: implementations/nestjs/harness/tests/fixtures/no-cross-aggregate-reference/good/src/payment/domain/refund.ts
export class Refund {
  public readonly refundId: string
  public readonly paymentId: string
  public readonly amount: number

  constructor(params: { refundId: string; paymentId: string; amount: number }) {
    this.refundId = params.refundId
    this.paymentId = params.paymentId
    this.amount = params.amount
  }
}

File: implementations/fastapi/harness/tests/fixtures/no-cross-aggregate-reference/bad-payment-references-refund/src/payment/domain/refund.py
from datetime import datetime


class Refund:
    def __init__(self, refund_id: str, payment_id: str, amount: int, created_at: datetime) -> None:
        self.refund_id = refund_id
        self.payment_id = payment_id
        self.amount = amount
        self.created_at = created_at
